src/composable/composable.ts

```diff
@@ -98,8 +105,8 @@ function all<T extends [Composable, ...Composable[]]>(...fns: T) {
     }
 
     return success((results as Success<any>[]).map(({ data }) => data))
-  }) as unknown as Composable<
-    (...args: Parameters<T[0]>) => {
+  }) as Composable<
```

Comment:
I have tried to extract `AllReturn` but this will prevent TS from showing us the full inline resolved type yielding a poorer DX. We can do this in the future in case we find out how to tell TS to inline resolved types.